.github/workflows/test.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
           - beta
           - nightly
           - 1.61.0 # MSRV - keep in sync with what rustls considers MSRV
-        os: [ubuntu-20.04]
+        os: [ubuntu-latest]
```

Comment:
I have a vague memory of proposing this switch early in my contributions to rustls-ffi and getting feedback that there was a preference to stay on 20.04 explicitly. I can't find the relevant PR/feedback now, so I could be remembering wrong.

@jsha Does this jog your memory? Do you have a preference here?